GeneralsMD/Code/GameEngine/Include/Common/CopyProtection.h

```diff
@@ -33,7 +33,7 @@
 #define COPYPROTECTION_H
 
 // Comment out the following line to disable copy protection checks
-#define DO_COPY_PROTECTION
+//#define DO_COPY_PROTECTION
```

Comment:
Might need to note this somewhere?

Reply:
Its noted in the commit notes that I disable it for ZH, its already commented out in the Generals branch so I guess its just an oversight from LFeenanEA or whoever prepared the final code drop?